Content.Server/Recycling/RecyclerSystem.cs

```diff
@@ -92,8 +92,8 @@ private void Recycle(RecyclerComponent component, EntityUid entity)
         {
             RecyclableComponent? recyclable = null;
 
-            // Can only recycle things that are recyclable... And also check the safety of the thing to recycle.
-            if (!_tags.HasTag(entity, "Recyclable") &&
+            // Can only recycle things that are trash or recyclable... And also check the safety of the thing to recycle.
+            if (!_tags.HasTag(entity, "Trash") &&
```

Comment:
Won't this make it stop working on stuff with the recyclable tag? Should probably be using hasanytag

Reply:
Yes. That’s what I said in the description.
“HasAnyTag” is useful, though - thanks.